src/sqlfluff/core/plugin/lib.py

```diff
@@ -1,33 +1,29 @@
 """Base implementation for the plugin."""
 
-import pluggy
 import os.path
+from sqlfluff.core.config import ConfigLoader
 from sqlfluff.core.plugin import hookimpl
+from sqlfluff.core.rules.config_info import STANDARD_CONFIG_INFO_DICT
+from sqlfluff.core.rules.std import get_rules_from_path
 
-class Plugin:
```

Comment:
Now that we're no longer calling `.register()`, the plugin implementation no longer needs to be enclosed in a class.